editor.planx.uk/src/@planx/components/shared/Schema/InputFields/ChecklistFieldInput.tsx

```diff
@@ -10,11 +10,8 @@ import { getFieldProps, Props } from ".";
 import { FieldInputDescription } from "./shared";
 
 export const ChecklistFieldInput: React.FC<Props<ChecklistField>> = (props) => {
-  const {
-    data: { title, description, options },
-    formik,
-  } = props;
-  const { id, errorMessage, name, value } = getFieldProps(props);
+  const { data, formik } = props;
+  const { id, errorMessage, name, value, title } = getFieldProps(props);
```

Comment:
Previously, we were reading `title` directly from props instead of passing through `getFieldProps` which is responsible for appending `(optional)` onto field titles. Checklist titles now displaying correctly: 
![Screenshot from 2025-04-30 10-27-06](https://github.com/user-attachments/assets/c2cfa656-c693-4ccd-be5e-ec5542c0a0a8)